Assets/Scripts/InventoryAndItemsScripts/Items/Collectables/InteractCollectable.cs

```diff
@@ -9,11 +9,11 @@ namespace Items.Collectables
 {
     public class InteractCollectable : Collectable, IInteractable
     {
-        [SerializeField] private int numberOfTimesCanInteract = 1;
+        [SerializeField] private int timesCanInteract = 1;
```

Comment:
would there ever be a time where this is more than 1 but less than infinity? seems weird if you could only interact with something like, 3 times

Reply:
This is something Saran added - the purple cube in the scene could be interacted with three times and then it would disappear. I'm not sure if we ever _need_ it to be more than 1/less than infinity...maybe if we have something that works like a coin block in Mario, where you can hit it multiple times before it disappears?